sdk-api-src/content/memoryapi/nf-memoryapi-setprocessvalidcalltargets.md

```diff
@@ -52,13 +52,9 @@ ms.custom: 19H1
 ## -description
 
 
-<p class="CCE_Message">[Some information relates to pre-released product which may be substantially modified before it's commercially released. Microsoft makes no warranties, express or implied, with respect to the information provided here.]
+<div class="alert"><b>Note</b>  Some information relates to pre-released product which may be substantially modified before it's commercially released. Microsoft makes no warranties, express or implied, with respect to the information provided here.</div><div> </div>
```

Comment:
Can we remove this? This API has been present since around Windows 8.1, it's stable at this point.

Reply:
Yes. Thanks for catching.